### PR TITLE
Shorten the menu command

### PR DESCRIPTION
**Pull request by blaggacao**
A shot into the _darknix_.

### CODE REVIEW

**Comment by 06kellyjac** [comment]
I'm not sure that the help command should change name so people who need help can find it in the same place.
Maybe it'd be better as `devshell-menu` + `${config.name}-menu`

**Comment by blaggacao** [comment]
> I'm not sure that the help command should change name so people who need help can find it in the same place.
Maybe it'd be better as devshell-menu + ${config.name}-menu

There are to aspects to it:
- alignment between projects
- semantical identity of rhw command

The problem with devshell-help is that it's context sensitive. So the same command does not result in the same output in different projects.

Looking at what the help command does, it is _nkt_ help for devshell. Rather it is help for the _projecr_ shell that happens to be construed with devshell.

If that really would be a problem, I would rather map it to `help` + `${config.name}-help`.

But then, to avoid cluttering the interface so early on in the project, I would rather prefer a trade off decision than having two commands:
So I'd pledge for either `devshell-help` or `${config.name}-help`. What ever is semantically more concise (I belive the latter).

File: options.nix [hunk 1]
```diff
@@ -191,7 +188,7 @@ in
     commands = [
       {
         help = "prints this menu";
-        name = "devshell-menu";
+        name = "${config.name}-menu";
```
Comment (zimbatm) [inline options.nix:191]: I'm not sure about that

Comment (blaggacao) [reply options.nix:191]: My reasoning was:

_Show me the help for that particular **instance** of devshell which is identified by the name of `${config.name}`_

On the contrary, it looses devshell branding.

Ultimately, this is akin to a marketing decision.

Comment (zimbatm) [reply options.nix:191]: It could also be renamed to `menu` so it's shorter to type.

Comment (blaggacao) [reply options.nix:191]: I agree and will adapt accordingly.

**Pull request merged**